Client.java

```diff
@@ -30,6 +30,12 @@ void connect(ClientUI clientUI) {
     void disconnect(ClientUI clientUI) {
         try {
             if (clientSocket != null) {
+                DataOutputStream outToServer = new DataOutputStream(clientSocket.getOutputStream());
+
+                String sendingSentence = "Disconnect\n";
```

Comment:
Here is this disconnect method, I would pass the message "Disconnect" which will then be read by the server saying that it is time to disconnect